src/components/contact/contact.scss

```diff
@@ -1,32 +1,20 @@
-@import "../../utils/variables";
-
 .contact-container {
-  width: 90%;
-  padding: $container-padding;
-  display: flex;
-  flex-direction: column;
-  justify-content: space-between;
-  font-size: $regular-font-size;
-  font-weight: $main-font-weight;
-  h1 {
-    font-weight: 400;
-  }
+  height: 100%;
+  padding: 15px;
 }
 ul {
-  height: 45vh;
+  list-style-type: none;
+  height: 65vh;
```

Comment:
Try not to use vh and vw to set height and width. Only App component should be set like that. You can use % for this.